Automattic/wp-calypso
# Taxonomies: Show taxonomies on all the available space on the page

Use WindowScroller from react-virtualized to show the taxonomies on all the available space on the page.

Also, I noticed that we can scroll on every page of Calypso even if the content fits in the available space, I fixed that here, by changing the margin to padding on `.layout__content` 

<img width="1280" alt="capture d ecran 2016-11-09 a 4 34 06 pm" src="https://cloud.githubusercontent.com/assets/272444/20143781/925ab5c0-a69a-11e6-8d64-8c28f9f9b957.png">


**Testing instructions**

 * Open the taxonomies page `/settings/taxonomies/$site/category`
 * Make sure to have a large number of categories
 * You can scroll smoothly and the taxonomies are shown on all the available space
